cranelift/2024/cranelift-10-16.md

```diff
@@ -17,5 +17,34 @@
 
 ### Attendees
 
+* Nick Fitzgerald
+* Alexa VanHattum
+* Ulrich Weigand
+* Andrew Brown
+
 ### Notes
 
+* Updates:
+  * Alexa: intel announced new x86 semantics they are releasing, hope to use
+    them in the near-ish future with our ISLE verification
+  * Ulrich: none
+  * Nick: none
+  * Andrew:
+    * thinking about rewriting the x64 assembler and encoder
+    * possibly adding APX extension in the future
+    * general discussion of describing instructions in a table, generating Rust
+      and ISLE code from it
+      * want to mechanically generate instruction encoding, regalloc
+        constraints, pretty printers, etc... from the table, put each
+        instruction's semantics in the table, etc...
+      * general consensus that generating mach insts in `cranelift-codegen-meta`
+        via builders is preferable, rather than parsing text and maybe requiring
+        new deps. parsing (and/or building new deps) would negatively impact our
+        build times
```

Comment:
nit: this wasn't a general consensus as much as it was a concerned opinion @fitzgen had; I think all options are open at this early point in time, though obviously we wouldn't want to slow down build times.

Reply:
maybe I misread the room! we can certainly reword the notes if you'd like

Reply:
Nah, it's fine... I just didn't want to decide on how to implement it just yet.